sample.env

```diff
@@ -126,7 +126,7 @@ CONNECT_CONFIG_RELATIVE_PATH="./jikkou/kafka-connectors-values.yaml"
 ### DEDUPLICATOR variables - START ###
 
 # Required for building the deduplicator. Documentation: https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-apache-maven-registry#authenticating-to-github-packages
-MAVEN_GITHUB_TOKEN=
+MAVEN_GITHUB_TOKEN=replace_me
```

Comment:
Suggest adding language to explain this must be replaced with a real token for dedup to work